Tools/BuildAndPackage.ps1

```diff
@@ -1,3 +1,8 @@
+[CmdletBinding()]
+Param(
```

Comment:
Let's put a TODO to confirm the value is correct. If we incorrectly use a lower version, we could publish bits that won't get updated correctly. Perhaps we can use the GitHub API to check for the latest release, or confirm we have the previous release on Cloud Storage, etc.